docs/src/operations/requirements.md

```diff
@@ -55,13 +63,17 @@ made.
 - CPU
   - 16 cores / 32 threads, or more
 - RAM
-  - 512 GB or more if `account-index` is used
+  - 512 GB or more if an `account-index` is used, 1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
```

Comment:
in 1.18 the secondary index backing will switch to `RwLock<HashMap>` from `DashMap`, which should bring total secondary index memory consumption back below 512GB. anecdotally, in the 250-300GB range

Reply:
Added note for 1.18, will update when that goes to mainnet-beta